include/functions.h

```diff
@@ -877,9 +877,9 @@ f32 Math_SmoothStepToDegF(f32* pValue, f32 target, f32 fraction, f32 step, f32 m
 s16 Math_SmoothStepToS(s16* pValue, s16 target, s16 scale, s16 step, s16 minStep);
 void Math_ApproachS(s16* pValue, s16 target, s16 scale, s16 step);
 void Color_RGBA8_Copy(Color_RGBA8* dst, Color_RGBA8* src);
-void func_80078884(u16 sfxId);
-void func_800788CC(u16 sfxId);
-void func_80078914(Vec3f* arg0, u16 sfxId);
+void Audio_PlaySfxGeneralWithDefaultSettings1(u16 sfxId);
+void Audio_PlaySfxGeneralWithDefaultSettings2(u16 sfxId);
+void Audio_PlaySfxGeneralWithDefaultSettingsAtPosition(Vec3f* arg0, u16 sfxId);
```

Comment:
Note this clashes with #1470 , and overall with PlaySfx functions our local resident audio person @engineer124 has a big plan for naming the *many* PlaySfx wrappers (see the comment on that PR)

Reply:
True, I haven't read this PR in more detail. I am fine with having there a second underscore. @engineer124 Nothing to do here, I will just adjust my code to yours if this is alright for all of you:

```suggestion
void Audio_PlaySfx_GeneralWithDefaultSettings1(u16 sfxId);
void Audio_PlaySfx_GeneralWithDefaultSettings2(u16 sfxId);
void Audio_PlaySfx_GeneralWithDefaultSettingsAtPosition(Vec3f* arg0, u16 sfxId);
```

It's frustrating that both functions "Audio_PlaySfx_GeneralWithDefaultSettings" do the same, look the same, but are not allowed to be optmized...